layouts/default.vue

```diff
@@ -1,19 +1,64 @@
 <template>
   <div class="app">
-    <div class="app__header">
+    <div :class="['app_header', {'app__header__open' : this.header }]">
       <h1 class="app__header__title">Unicorns</h1>
-      <back-button v-if="!isHomepage" />
+      <back-button v-if="!isHomepage" @close-menu="closeMenu"/>
+      <menu-bouton @action-menu="actMenu"/>
+      <div class="app__header__menu-title" v-if="isOpen">
+       <item-menu v-for="(items,index) in itemsMenu" :item="items" :key="index" @nav-menu="navMenu"/>
+      </div>
     </div>
     <Nuxt />
+
+    <fab-add v-if="isHomepage" @close-menu="closeMenu"/>
   </div>
 </template>
 <script>
 import BackButton from "@/components/BackButton";
+import MenuBouton from '@/components/MenuBouton.vue';
+import ItemMenu from '@/components/ItemMenu.vue';
+import FabAdd from '@/components/FabAdd.vue';
 export default {
-  components: {BackButton},
+  components: {BackButton, MenuBouton, ItemMenu, FabAdd},
+
+  async fetch(){
+    let array = this.$http.$get(process.env.baseUrl + '/navigations?_sort=ordre').then(array => { return array })
```

Comment:
ici pourquoi utiliser let ? Tu ne réaffectes pas une autre valeur à la variable array, donc tu peux directement faire un this.itemsMenu =
Tu peux également mettre directement un await devant l'appel : this.itemsMenu = await this.$http.$get()